libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -288,7 +292,18 @@ AP_GPS_SBF::process_message(void)
 
         state.hdop = last_hdop;
     }
+    // ReceiverStatus
+    if (blockid == 4014) {
```

Comment:
It should be, I admit there were a number of slightly different style conventions I thought aligned better with master/other drivers but since I wasn't walking over the entire driver I ignored them. I added else's everywhere here, but I suspect:
1. The compiler might have been doing the right thing here anyways, (it's a pretty obvious one from the compiliers point of view if the first is true then the rest can't be, but I didn't check the output).
2. This should really be a switch on blockid rather then a if/else if structure....

Anyways in the spirit of minimal changes I just added the else/if's where needed.